src/gebt_poc/solver.cpp

```diff
@@ -16,22 +16,46 @@ UserDefinedQuadrature::UserDefinedQuadrature(
 }
 
 void Interpolate(
+    Kokkos::View<double*> nodal_values, Kokkos::View<double*> interpolation_function,
+    Kokkos::View<double*> interpolated_values
+) {
+    const auto n_nodes = nodal_values.extent(0) / kNumberOfLieAlgebraComponents;
+    KokkosBlas::fill(interpolated_values, 0.);
+    for (std::size_t i = 0; i < n_nodes; ++i) {
+        auto index = i * kNumberOfLieAlgebraComponents;
+        KokkosBlas::axpy(
+            interpolation_function(i),
```

Comment:
We can't access interpolation_function on host like this.  We either need to copy interpolation_function to host or use another method to keep data on device.